website_sale_product_attribute_value_filter_existing/tests/test_website_sale_product_attribute_value_filter_existing.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+from odoo.tests.common import HttpCase
+
+
+class WebsiteSaleHttpCase(HttpCase):
```

Comment:
@anothingguy  i face a warning when running the tests:
`2023-03-17 08:24:38,348 44513 WARNING v16c_website_sale_product_attribute_value_filter_existing odoo.addons.website_sale_product_attribute_value_filter_existing.tests.test_website_sale_product_attribute_value_filter_existing.WebsiteSaleHttpCase: HttpCase test should be in post_install only `

Reply:
i didn't notice this warning, but it seems the warning doesn't make sense, this is tests case of website, so it should be in here, not `post_install`

Reply:
HttpCase that use `browser_js` need to run at `post_install` (`@odoo.tests.common.tagged('post_install', '-at_install')`)

Reply:
ah ok @nilshamerlinck , I got it

Reply:
updated